lib/CMakeLists.txt

```diff
@@ -74,6 +74,8 @@ find_program(CCACHE ccache)
 if(CCACHE)
   set(CMAKE_CXX_COMPILER_LAUNCHER ${CCACHE})
 endif()
+message(STATUS "Tried to find ccache, found:")
+message(STATUS ${CCACHE})
```

Comment:
What about these? Are they left over from debugging?

Reply:
Yes, but I decided to keep them anyway, unless you mind?

Reply:
No, just curious.